src/Models/Servers/Servers.php

```diff
@@ -126,8 +128,13 @@ public function createInDatacenter(
         $startAfterCreate = true,
         $user_data = '',
         $volumes = [],
-        $automount = false
+        $automount = false,
+        $networks = []
     ): APIResponse {
+        $networkIds = [];
+        foreach ($networks as $network) {
+            $networkIds[] = $network->id;
```

Comment:
The call should just work with ids like it does with the volume IDs. Not with the whole objects.